services/spar/test-integration/Util/Scim.hs

```diff
@@ -145,6 +145,21 @@ randomScimUserWithSubjectAndRichInfo richInfo = do
       subj
     )
 
+randomScimUserWithEmail :: MonadRandom m => m (Scim.User.User SparTag, Email)
+randomScimUserWithEmail = do
+  suffix <- cs <$> replicateM 7 (getRandomR ('0', '9'))
+  let email = Email ("email" <> suffix) "example.com"
+      externalId = fromEmail email
+  pure
+    ( (Scim.User.empty userSchemas ("scimuser_" <> suffix) (ScimUserExtra (RichInfo mempty mempty)))
+        { Scim.User.displayName = Just ("Scim User #" <> suffix),
+          Scim.User.externalId = Just externalId,
+          Scim.User.emails = mempty,
```

Comment:
Aren't these fields already set to `mempty`? why have them here? can be removed right?

Reply:
4790d35d4e5ac3bb52b329172a5db4a46e81783